src/main/java/de/medizininformatikinitiative/torch/management/StructureDefinitionHandler.java

```diff
@@ -70,7 +71,7 @@ public boolean known(String profile) {
      * @return The StructureDefinition corresponding to the base URL (ignores version)
      */
     public Optional<CompiledStructureDefinition> getDefinition(String url) {
-        return getDefinition(Set.of(url));
+        return getDefinitions(Set.of(url)).stream().findFirst();
```

Comment:
I don't like this approach because it's not clear that you can simply pick the first definition. The fact that you supply only one URL is not obvious. Please implement the method directly without using `getDefinitions`.